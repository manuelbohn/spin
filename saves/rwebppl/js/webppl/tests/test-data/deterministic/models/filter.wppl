[
  filter(isEven, []),
  filter(isEven, [0, 1, 2, 3, 4, 5, 6, 7, 8, 9]),
  filter(isOdd, [0, 1, 2, 3, 4, 5, 6, 7, 8, 9]),
  sum(filter(isEven, mapN(function(i) { return i }, 50000))),
  sum(filter(isOdd, mapN(function(i) { return i }, 50000))),
  find(isEven, []) === undefined,
  find(isEven, [1, 3, 5, 7, 9, 2, 4]),
  remove(10, mapN(idF, 11))
]
