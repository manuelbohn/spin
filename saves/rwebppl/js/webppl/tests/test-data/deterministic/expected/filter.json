{
  "result": [
    [],
    [0,2,4,6,8],
    [1,3,5,7,9],
    624975000,
    625000000,
    true,
    2,
    [0,1,2,3,4,5,6,7,8,9]
  ]
}
